include/Background.h

```diff
@@ -4,11 +4,10 @@
 #include "Access.h"
 #include "GameComponents.h"
 
-
 struct BackgroundSystem
-	: public ecs::System
-	, public MutAccessGroupStorage<Background,KeyBindings, Position, Sprite>
-	, public MutAccessUnique<OurGlobalVar>
+    : public ecs::System,
+      public MutAccessGroupStorage<Background, KeyBindings, Position, Sprite>,
+      public MutAccessUnique<OurGlobalVar>
```

Comment:
These should also be tabs and commas at the start to make it consistent.
`class AssetModule` in `Assets.h` is a good example.